openhantek/src/hantekdso/controlsettings.h

```diff
@@ -51,6 +51,7 @@ struct ControlSettings {
     ~ControlSettings();
     ControlSettingsSamplerate samplerate;        ///< The samplerate settings
     std::vector<ControlSettingsVoltage> voltage; ///< The amplification settings
+    std::vector<double> probeGain; ///< The gain probe gain for the current channel
```

Comment:
I would prefer a probeIndex/probeId here

Reply:
With probeIndex  you mean the index of the gain in the list of possible gain of a probe?

Reply:
I would keep this a single gain to avoid the overhead of accessing an array to get the prove gain during the sample acquisition.